cmd/weaver/main.go

```diff
@@ -74,11 +74,25 @@ func main() {
 	switch flag.Arg(0) {
 	case "generate":
 		generateFlags := flag.NewFlagSet("generate", flag.ExitOnError)
+		tags := generateFlags.String("tags", "not specified", "Optional tags for the generate command")
 		generateFlags.Usage = func() {
 			fmt.Fprintln(os.Stderr, generate.Usage)
 		}
 		generateFlags.Parse(flag.Args()[1:])
-		if err := generate.Generate(".", flag.Args()[1:], generate.Options{}); err != nil {
+		buildTags := "--tags=ignoreWeaverGen"
```

Comment:
See below. I think we should just have "ignoreWeaverGen" here and supply --tags= prefix later.

Reply:
Dobe